docs/writing-stories/index.mdx

```diff
@@ -126,17 +223,26 @@ components/
 
 ### Rename stories
 
-You can rename any particular story you need. For instance, to give it a more accurate name. Here's how you can change the name of the `Primary` story:
+By default, Storybook uses the name of the story export as the basis for the story name. However, you can customize the name of your story by adding a `name` property to the story object. This is useful when you want to provide a more descriptive or user-friendly name for your story.
+
+<If renderer="svelte">
+
+If you're using Svelte CSF, the `name` property is usually fairly descriptive and user-friendly already, so there is little need for renaming. However, the `name` is used as the basis for the export name, which must be unique within a file. In rare cases, this can lead to naming conflicts. For example, stories with the names "Primary" and "Primary!" would both be transformed to the "Primary" export name. To avoid this, you can use the `exportName` property to specify a unique export name for your story.
```

Comment:
I think it could make sense to document here, that `exportName` is also useful for when you want your story to have an easy-to-know export, when you need to import the story in MDX or other stories.

Eg. if you have this story that you want to reference MDX:

```svelte
<Story name="My arg-less story!"/>
```

It would be pretty hard to know how to import this (I wouldn't even know without looking behind-the-scenes)

```mdx
import { whatName? } from './my.stories.svelte';
```

However if you set an explicit export name, it becomes clear:

```svelte
<Story
  name="My arg-less story!"
  exportName="ArgLess"
/>
```

```mdx
import { ArgLess } from './my.stories.svelte';
```